usr/share/rear/verify/BAREOS/default/050_check_requirements.sh

```diff
@@ -18,17 +20,46 @@ if [ "$BEXTRACT_DEVICE" -o "$BEXTRACT_VOLUME" ]; then
 
 else
 
-   ### Bareos support using bconsole
-   [ -x /usr/sbin/bareos-fd ]
-   StopIfError "Bareos executable (bareos-fd) missing or not executable"
+    ### Bareos support using bconsole
+
+    if ! has_binary bareos-fd; then
+        Error "Bareos executable (bareos-fd) missing or not executable"
+    fi
+
+    if ! bareos-fd -t; then
+        Error "bareos-fd: configuration invalid"
+    fi
+
+    if ! systemctl --quiet is-active bareos-fd.service; then
+        Log "$(systemctl status bareos-fd.service)"
+        Error "bareos-fd.service is not running"
+    fi
+
+    if ! has_binary bconsole; then
+        Error "Bareos executable (bconsole) missing or not executable"
+    fi
+
+    if ! bconsole -t; then
+        Error "Bareos bconsole configuration invalid"
+    fi
+
+    LogPrint "Connecting to the Bareos Director ..."
+    local bconsole_client_status=$(bconsole <<< "status client=$BAREOS_CLIENT")
+    local rc=$?
+    Log "${bconsole_client_status}"
+    if [ $rc -ne 0 ]; then
+        Error "Failed to connect to Bareos Director."
```

Comment:
Please also include the output of `bconsole` so that users don't have to open the logfile.